cognite/neat/_data_model/validation/dms/_reverse_connection_validators.py

```diff
@@ -0,0 +1,346 @@
+from dataclasses import dataclass
+from itertools import chain
+
+from pyparsing import cast
+
+from cognite.neat._data_model.models.dms._container import ContainerRequest
+from cognite.neat._data_model.models.dms._data_types import DataType, DirectNodeRelation
+from cognite.neat._data_model.models.dms._references import (
+    ContainerDirectReference,
+    ContainerReference,
+    ViewDirectReference,
+    ViewReference,
+)
+from cognite.neat._data_model.models.dms._view_property import ViewCorePropertyRequest
+from cognite.neat._data_model.models.dms._views import ViewRequest
+from cognite.neat._data_model.validation.dms._base import DataModelValidator
+from cognite.neat._issues import ConsistencyError, Recommendation
+
+
+@dataclass
+class ReverseConnectionContext:
+    """Context for validating a bidirectional connection."""
```

Comment:
Could you extend the docstring with the arguments? I can guess, but it would nice to know exactly what each of these fields mean